create-pull-request.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+/**
+ * Small Utility script to help us creating a Pull Request using Github's Rest API.
+ */
+
+$user = getenv('GITHUB_ACTOR');
+$token = getenv('GITHUB_TOKEN');
+$repository = getenv('GITHUB_REPOSITORY');
+$body = file_get_contents('php://stdin');
+
+['h' => $head] = getopt('h:');
+
+if (empty($user)) {
+    throw new RuntimeException('GITHUB_ACTOR not set!');
+}
+
+if (empty($token)) {
+    throw new RuntimeException('GITHUB_TOKEN not set!');
+}
+
+if (empty($repository)) {
+    throw new RuntimeException('GITHUB_REPOSITORY not set!');
+}
+
+if (empty($body)) {
+    throw new RuntimeException('Update message empty!');
+}
+
+if (empty($head)) {
+    throw new RuntimeException('Head branch not specified!');
+}
+
+$ch = curl_init();
+curl_setopt($ch, CURLOPT_URL, "https://api.github.com/repos/$repository/pulls");
+curl_setopt($ch, CURLOPT_POST, 1);
+curl_setopt($ch, CURLOPT_USERAGENT, 'Drupal7-Security-Updates');
+curl_setopt($ch, CURLOPT_USERPWD, "$user:$token");
+curl_setopt($ch, CURLOPT_POSTFIELDS, json_encode(
+  [
+    'title' => 'Automated Security Updates',
```

Comment:
Can we add the tag name to the title? 🐶

Reply:
I thought a bit about this and I think it's not worth it. In order to be able to do that, I'd have to either pass an additional param to the script or extract it from the branch name. Besides this, I kept thinking on what difference it would really make to have the tag in title and I couldn't think of any. In the end, the important thing is to know that we have security updates available and that they should be merged as quickly as possible.

Anyway, I have followed your suggestion to add the tag number to the note in the Pull Request, so I guess that would be a good enough way for people to know which tag was used?